src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.util.Collections;
+import java.util.List;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@DisplayName("메뉴그룹 기능 관리")
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    private MenuGroupService menuGroupService;
+    private MenuGroup 두마리치킨메뉴;
+
+
+    @BeforeEach
+    void setUp() {
+        두마리치킨메뉴 = 메뉴그룹_생성("두마리치킨메뉴");
+        menuGroupService = new MenuGroupService(menuGroupDao);
+    }
+
+
+    @Test
+    @DisplayName("`메뉴그룹`을 등록할 수 있다.")
+    void 메뉴그룹_등록() {
+        // given
+        when(menuGroupDao.save(any())).thenReturn(두마리치킨메뉴);
```

Comment:
BDD(Behavior Driven Development) 스타일의 테스트 작성방법을 사용해서 given/when/then 으로 테스트를 작성해주셨는데요~

`stubbing` 을 BDDMockito.given()을 사용해보면 어떨까요?

```suggestion
        given(menuGroupDao.save(any())).willReturn(두마리치킨메뉴);
```

참고: https://velog.io/@lxxjn0/Mockito%EC%99%80-BDDMockito%EB%8A%94-%EB%AD%90%EA%B0%80-%EB%8B%A4%EB%A5%BC%EA%B9%8C

Reply:
감사합니다!